CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 cmake_minimum_required(VERSION 3.16)
 project(ArborX CXX)
 
-find_package(Kokkos 3.4 REQUIRED)
+find_package(Kokkos 3.4 REQUIRED CONFIG)
+message(STATUS "Found Kokkos: ${Kokkos_VERSION}")
```

Comment:
```suggestion
message(STATUS "Found Kokkos: ${Kokkos_VERSION} at ${Kokkos_DIR}")
```

Reply:
Maybe follow Boost then? Like
```
-- Found Kokkos: /home/xap/local/opt/kokkos-serial-56468253e/lib/cmake/Kokkos (found suitable version "3.5.99", minimum required is "3.4.00")
```

Reply:
Sounds good.  I was actually trying to figure out where this message is emitted from (the Boost one)

Reply:
You didn't quite apply the suggestion you made above.  Is that an oversight?

Reply:
I did not see your reply. I'm fine either way. I now realize that each package has its own style. For example, MPI has a different way to print them out compared to Boost. So there is not going to be a single consistent style anyway.

Reply:
@dalg24 Are you OK with the current version?